documents/sql_scripts/CVManager_CreateTables.sql

```diff
@@ -1,4 +1,6 @@
 -- RSU Manager Cloud Run Tables
+CREATE EXTENSION postgis;

```

Comment:
Is this needed with already pulling the postgis image?

Reply:
It is. I tried it without, but the database didn't know what a geometry object was until I created the extension.